### ISSUE TITLE
[Feature request] Individual cvars for features currently under r_softemu

### ISSUE THREAD

**Issue by h4724** [andrei-drexler/ironwail#74]
I mentioned in [#33](https://github.com/andrei-drexler/ironwail/issues/33#issuecomment-1036954206) that r_softemu actually handles a lot of different effects, and I would like to be able to control them individually instead of just using presets for all of them, given how much people's tastes with these things vary. By my count, you've got palletised lighting, texture space dithering, screen space dithering, UV distortion, and menu background all under one cvar currently.

I don't suggest removing r_softemu nor the combined menu option, because I'm sure some people find the generalised presets helpful, but it would be nice to have some finer control, e.g. for people who want texture space dithering but not screen space.

**Comment by OpenRift412** [comment]
Agreed, having a var for menu background specifically would be nice as well, since the 1.06-style background of r_softemu 3 still has some issues, it'd be nice to have it use the 1.08/winquake-style background used in r_softemu 2.

**Comment by gilabogus** [comment]
Interestingly:

Floor is rendered closer to the original software renderer in r_softemu "1" (but the walls aren't and there's some dithering everywhere)
![r_softemu_1](https://user-images.githubusercontent.com/103505513/168444355-4ff504ef-2c06-4b0e-ad26-ee361b2cf893.png)

The walls (well, lightmap shading) is more correct in r_softemu "3", but floor is "wrong"
![r_softemu_3](https://user-images.githubusercontent.com/103505513/168444378-f1076236-2fb2-4d21-b17c-38e22b96b0c4.png)

**Comment by KillPixelGames** [comment]
+1 for this.  having these options separated would be excellent.
